framework/source/class/qx/util/ResourceManager.js

```diff
@@ -291,8 +298,16 @@ qx.Class.define("qx.util.ResourceManager",
           }
           // If the resourceUri begins with a single slash, include the current
           // hostname
-          else if (resourceUri.match(/^\//) != null) {
-            statics.__urlPrefix[lib] = window.location.protocol + "//" + window.location.host;
+          else if (resourceUri.match(/^\//) != null)
```

Comment:
Please don't get into the political argument of where the braces go. qooxdoo is inconsistent, and we have differing views on the *one true way*. When changing an existing file, just try to remain as consistent as possible with the style already used in that file.